example/lib/helpers.js

```diff
@@ -36,7 +36,7 @@ function generateToken(req, GUID, opts) {
   // By default, expire the token after 7 days.
   // NOTE: the value for 'exp' needs to be in seconds since
   // the epoch as per the spec!
-  var expiresDefault = Math.floor(new Date().getTime()/1000) + 7*24*60*60;
+  var expiresDefault = Math.floor(Date.now() + (1000 * 60 * 60 * 24 * 7));
```

Comment:
`expiresIn` is defined in _seconds_ not Milliseconds. 
see: https://www.npmjs.com/package/jsonwebtoken
![image](https://user-images.githubusercontent.com/194400/31538677-3d5f6610-affe-11e7-8d4d-84bf8ea30aed.png)
and:
![image](https://user-images.githubusercontent.com/194400/31538870-d18d23e0-affe-11e7-8acc-a21e330696b0.png)